ted/src/main/java/frc/robot/common/DriveTrainConfig.java

```diff
@@ -0,0 +1,109 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Crescendo 2024
+// File: DrivetrainConfig.java
+// Intent: Forms a type to hold drivetrain config
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.common;
+
+import frc.robot.swerveLib.ModuleConfiguration;
+
+/**
+ * Forms a type to hold drivetrian config
+ */
+public class DrivetrainConfig {
```

Comment:
So casing on the Java file name is DriveTrainConfig and the class name is DrivetrainConfig.  One of these should be changed.